addon/components/pix-input-password.hbs

```diff
@@ -27,6 +29,7 @@
       aria-label={{this.ariaLabel}}
```

Comment:
celui la ne devrait pas y etre s'il y a un label plus haut
aria-label={{unless this.label this.ariaLabel}}